pilot/pkg/networking/plugin/authz/test_helper.go

```diff
@@ -189,3 +190,73 @@ func generatePolicyWithHTTPMethodAndGroupClaim(methodName, claimName string) *po
 		}},
 	}
 }
+
+// nolint:deadcode
+func generateExpectRBACForSinglePolicy(serviceRoleID string, rbacPolicy *policy.Policy) *policy.RBAC {
+	return &policy.RBAC{
+		Action: policy.RBAC_ALLOW,
+		Policies: map[string]*policy.Policy{
+			"service-role-" + serviceRoleID: rbacPolicy,
```

Comment:
I suggest we don't hard-code the name here, let the caller to put the full name when calling. This makes the code more clear and self-explanation. And it handles if we want to use a different name.

Reply:
Done!